File: lab2 /task3.py
n = int(input("Enter a positive integer between 3 and 9 inclusive: "))

while n < 3 or n > 9:
    print("Invalid number. Please enter a number between 3 and 9.")
    n = int(input("Please enter a number between 3 and 9: "))

for i in range(1, 2 * n):
    k = n - abs(n - i)

    for j in range(k):
        print("*", end=" ")

    print()

File: lab2 /task3.cpp
#include <iostream>
using namespace std;

int main() {
    int n;

    cout << "Please enter a number between 3 and 9: ";
    cin >> n;


    while (n < 3 || n > 9) {
        cout << "Invalid input. Please enter a number between 3 and 9: ";
        cin >> n;
    }


    for (int i = 1; i <= n; i++) {
        for (int j = 1; j <= i; j++) {
            cout << "*";
        }
        cout << endl;
    }

     for (int i = n - 1; i >= 1; i--) {
        for (int j = 1; j <= i; j++) {
            cout << "*";
        }
        cout << endl;
    }

    return 0;
}
